src/helpers/oracleClient.ts

```diff
@@ -2,18 +2,47 @@ import { oracle } from "@uma/sdk";
 import config from "constants/oracleConfig";
 import Events from "events";
 
+type OracleType = oracle.types.state.OracleType;
+type State = oracle.types.state.State;
+type Client = oracle.client.Client;
+
+export const isSupportedOracleType = oracle.utils.isSupportedOracleType;
+
 export { oracle };
+
+export type { OracleType, State, Client };
 export const events = new Events();
-export const client = oracle.client.factory(
-  config,
-  (state: oracle.types.state.State) => events.emit("change", state)
+export const clients = oracle.factory(
+  // the configs are an array of oracle configs, the order of the configs determine the order of clients returned
+  Object.fromEntries(config),
+  (oracleType: OracleType, state: State) =>
+    events.emit("change", oracleType, state)
```

Comment:
event emitter on the client has changed to include the oracle type at which the state change originates